src/start_location.cpp

```diff
@@ -446,3 +446,27 @@ void start_location::handle_heli_crash( player &u ) const
         }
     }
 }
+
+static void add_monsters( const tripoint &omtstart, const mongroup_id &type, float expected_points )
```

Comment:
Not necessary, but the file has an anonymous namespace already. I'd use it instead of `static`.